### DISCUSSION TITLE
我的 2016 盘点 — 码志

### DISCUSSION THREAD

**Discussion by mzlogin** [mzlogin/blog-comments#228] [Announcements]
https://mazhuang.org/2017/01/01/my-2016/

**Comment by chinakevinguo** [comment]
asdfadf

**Comment by yangtianrui95** [comment]
写的很好,加油

**Comment by mzlogin** [comment]
@yangtianrui95 谢谢！加油加油~

**Comment by yujie24** [comment]
有点儿期待你的2017总结

**Comment by mzlogin** [comment]
@yujie24 嗯嗯\~过几天估计也写一个哈哈\~

看了一眼你博客，现在样式有点问题，_config.yml 里的 url 的 http 改成 https 应该就好了。
